docs/reference/glossary.asciidoc

```diff
@@ -447,6 +447,18 @@ See the {ref}/indices-rollover-index.html[rollover index API].
 // end::rollover-def[]
 --
 
+ifdef::permanently-unreleased-branch[]
+
+[[glossary-rollup]] rollup ::
+// tag::rollup-def[]
+Aggregates an index's time-series data and stores the results in another index,
+called a rollup index. For example, you can roll up hourly data into daily or
+weekly summaries.
```

Comment:
I plan to add an xref to the updated overview page once that's ready. I've added a note to #65515.

I don't think that's a blocker for this PR tho.